common/scripted_triggers/wc_maa_triggers.txt

```diff
@@ -219,7 +218,7 @@ has_aerial_maa_trigger = {
 		this = culture:scourge
 	}
 }
-has_massive_aerial_maa_trigger = {
```

Comment:
Reminding you still use this trigger.

Reply:
Why `has_special_massive_aerial_maa_trigger` has more cultures than `has_massive_aerial_maa_trigger` when `has_special_massive_aerial_maa_trigger` is a special case of `has_massive_aerial_maa_trigger`.

Reply:
Fixed the way we fixed giant maa trigger, will it work even though we use different syntax for the tiggers?

One has "scope:culture = culture:dragonmaw" the other has "this = culture:scourge"